crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -136,6 +136,13 @@ pub struct Entity {
     index: u32,
 }
 
+impl MapEntities for Entity {
```

Comment:
I wouldn't add this implementation and just use `entity_mapper.map(entity)` instead of `entity.map_entities(entity_mapper)`. It's better to have a single way of doing it.

Reply:
Yep.

Reply:
Hm maybe others have an opinion? I personally prefer calling `map_entities` directly on `Entity`

Reply:
I think it's a wash, so I'm going to vote in favor of less code.